release/models/interfaces/openconfig-if-ip.yang

```diff
@@ -480,6 +486,16 @@ revision "2023-06-30" {
        "RFC 791: Internet Protocol";
     }
 
+   leaf gratuitous-arp-accepted {
+     type boolean;
+     default false;
+     description
+        "When set to true, gratuitous neighbor advertisements
```

Comment:
gratuitous ARPs (NA/ND terminology is specific to v6)

Reply:
Done